doc/ca65.sgml

```diff
@@ -652,7 +652,54 @@ problem in most cases.
 
 To force a specific order of evaluation, parentheses may be used, as usual.
 
+<sect>String Constants and Raw-String Constants<label id="String_Constants"><p>
```

Comment:
It should be a `<sect1>`.  It should be called `<sect1>String and Raw-String formats<p>`.  It should be put between `<sect1>Number format<p>` (line 504) and `<sect1>Conditional assembly<p>` (line 512).